Master

aaronwalton61 opened the pull request.

isharq committed:
Merge pull request #2 from slugmobile/master

Added hardware directory & update information

nullboundary committed:
Merge pull request #4 from slugmobile/master

Fixed a bug on bulbramp start/end menus

isharq committed:
Correcting spelling of Triggertrap

In the boards file, Triggertrap was spelled incorrectly - whoops.

catchin committed:
fixed case - important on Linux where files are case sensitive

catchin committed:
removed unneccessary executable bit

catchin committed:
replaced deprecated prog_char with char PROGMEM

catchin committed:
fixed build error: "const char PROGMEM * XY" are now "const char PROGMEM * const XY"

nullboundary committed:
Merge pull request #5 from catchin/master

Bugfixes for newer compiler versions and Linux

isharq committed:
Update libraries/TriggerTrap/TTConfig.h

Updated the reference to the laser sensor - was A3, should be A4.

ascorbic committed:
Update to 0.3.36

ascorbic committed:
Merge pull request #6 from ascorbic/master

Update to 0.3.36

ascorbic committed:
Update README to 0.3.36

ascorbic committed:
Add binaries, because GitHub has deprecated downloads.

ascorbic committed:
Merge branch 'master' of github.com:TriggerTrap/Triggertrap

ascorbic committed:
Add gitattributes

ascorbic committed:
Remove corrupted hexes

nullboundary commented:
Hi, yes thanks, this should probably be merged, but I am curious why you would want to merge this rather then just use the triggertrap branch?

isharq committed:
Commit schematics to Github

schematics used to live on http://triggertrap.com/schematics, but as
the site went down, the schematics went with them.
